.github/actions/deps/external/action.yml

```diff
@@ -65,6 +65,22 @@ runs:
         echo >> $GITHUB_PATH "$PATH"
       shell: bash
 
+    # nrf
+    - name: Get nrfutil 7+
+      if: inputs.platform == 'nrf'
+      run: |
+        wget https://developer.nordicsemi.com/.pc-tools/nrfutil/x64-linux/nrfutil
```

Comment:
I don't think that gets us anything because the next step is to "install" other commands. If they wanted it cached, then they should have packaged it in a standard way.